com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -286,7 +286,6 @@ internal void AddQueueItemToInboundFrame(QueueItemType qItemType, float timeStam
 
             //Write the packed version of the queueItem to our current queue history buffer
             queueHistoryItem.queueWriter.WriteUInt16((ushort)qItemType);
-            queueHistoryItem.queueWriter.WriteUInt16((ushort)0);
```

Comment:
@NoelStephensUnity 
this one was super dodgy!
the fact that we write stuff like this in `RpcQueueContainer` but read from `QueueHistoryFrame` on the other side makes things very hard to follow and debug.

while working on this removal work, I actually realized that we might potentially consider refactoring `RpcQueue` stuff in the future — or simply leave it as is up until more job-system/native-collections backed future (because we'll refractor/rewrite lots of things once we start to go that direction anyway `¯\_(ツ)_/¯`)